content/modules/add-module-to-database.md

```diff
@@ -43,3 +43,84 @@ To add modules to a database:
 
 1. Click **Show advanced options** and enter a port number for the database, such as: **12543**.
 1. Click **Activate**.
+
+
+## Upgrading the module for the database
+
+{{< note >}}
+After you upgrade the module for a database, the database shards are restarted.
+This causes a short interruption in the availability of this database across the cluster.
+{{< /note >}}
+
+To upgrade a module that is installed on a cluster:
+
+1. Connect to the terminal of a node in the cluster
+1. Run `rladmin status` to list the databases on the node.
+1. Copy the name of the database that uses the module that you want to upgrade.
+
+    ![rladmin_status-1](/images/rs/rladmin_status-1.png?width=1000&height=214)
+
+1. Find the exact module name and version:
+
+    1. Extract the module archive (zip) file.
+    1. Open the JSON file.
+    1. Find the module name and version number in the file.
+
+    An example of the JSON file for the RediSearch module is:
+
+    ![module_info-1](/images/rs/module_info-1.png?width=1000&height=382)
+
+1. To see the versions of the modules on the cluster, run either:
+
+    - `rladmin status modules` - Shows the latest modules available on the cluster
+        and the modules used by databases.
+    - `rladmin status modules all` - Shows all of the modules available on the cluster
+        and the modules used by databases.
+
+1. To upgrade a database to the latest version of Redis
```

Comment:
If they can upgrade the database and its modules all at once, why would they need the previous few steps to find the module name and version?

Reply:
to make sure you upgrade to the module version you wanted to